helm/charts/determined/values.yaml

```diff
@@ -19,6 +19,11 @@ masterPort: 8080
 # load-balancer.
 useNodePortForMaster: false
 
+# Enable route support for Openshift by setting enabled to true. Configure tls termination (i.e edge) if needed.
+# route:
```

Comment:
Mostly this looks good. I lack any context on exactly how we would test this, etc. but it's a no-op if you don't enable it, so it's not going to cause any regressions, etc.

I do feel like we ought to namespace this under "openshift" somehow since it's an OpenShift-specific API that it's enabling the use of. Maybe openShiftRoute: instead of route: Thoughts?

Reply:
that makes sense. I will change it for openshiftRoute